tidb-cloud/cli-reference.md

```diff
@@ -0,0 +1,44 @@
+---
+title: TiDB Cloud CLI Reference
+Summary: The reference of the TiDB Cloud CLI.
+---
+
+# TiDB Cloud CLI Reference
+
+Use the TiDB Cloud CLI to manage your TiDB Cloud clusters, import data to your clusters, and more from your terminal.
+
+## Getting Started
+
+Make sure to first [set up your TiDB Cloud CLI environment](/tidb-cloud/get-started-with-cli.md). Once you installed the `ticloud` CLI, you can use it to manage your TiDB Cloud clusters from the command line.
+
+## Available Commands
+
+Use `ticloud [command] [subcommand]` to start up the `ticloud` CLI in your terminal. For users using TiUP, use `tiup cloud [command] [subcommand]` instead.
+
```

Comment:
Question: For users who have installed TiUP, if they run `ticloud [command] [subcommand]`, will it work?

Reply:
It doesn't work.